docs/README.md

```diff
@@ -39,6 +39,7 @@ Now, you can use all the blocks exported by the `product-price` app. Check out t
 | `product-list-price-range`    | Renders the product list price range. It follows the same logic applied to the `ListPrice`: if its value is equal to the product selling price, this block is not rendered. | 
 | `product-selling-price-range` | The product selling price range. | 
 | `product-seller-name` | Renders the name of the product's seller. |
+| `product-price-suspense` | When the price is loading, it will render a fallback component; otherwise, it will render its children. It is extremely useful when the store works with async prices. |
```

Comment:
```suggestion
| `product-price-suspense` | Renders a loading spinner or its children blocks when the product price is loading i.e. not rendered yet. It is extremely useful when the store works with async prices. |
```

Reply:
I think that's not what I mean.
When the price is loading, this component will render the `Fallback` component.
When the price is not loading, it will render its children.

It's the same behavior as [React Suspense](https://pt-br.reactjs.org/docs/concurrent-mode-suspense.html).

From the way it is written, it looks like both can be rendered while the price is loading.

Reply:
Perfect! Now I understand it! In practice (on the UI), what would be the fallback component?

Reply:
You can pass any IO component to the `Fallback` prop, but the default value is a Spinner